recipe/meta.yaml

```diff
@@ -58,17 +58,17 @@ requirements:
     - azure-identity-cpp
     - azure-storage-blobs-cpp
     - bitmagic
-    - krb5 =1.20.1
+    - krb5
     - xxhash
     - semimap
     - grpcio-tools
     - protobuf
     - spdlog
     - robin_hood
     - recycle
-    - lmdb ==0.9.22
+    - lmdb
```

Comment:
Another hiccough in the pinning repo, fixed by https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4928

Reply:
I would be in favor to also unpin, but the newer versions of LMDB have slight changes of behavior compared to this one.

Reply:
In https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4930 you were saying just now that the difference between 0.9.x and 0.9.x shouldn't matter?

Reply:
Updating to the latest version of ArcticDB makes 1 test for ArcticDB fail and apparently this is not acceptable, see: https://github.com/man-group/ArcticDB/pull/437

Reply:
Updating to the latest version of ArcticDB makes 1 test for ArcticDB fail, see: https://github.com/man-group/ArcticDB/pull/437

Reply:
LMBD is less critical because it has no other dependencies itself, so if you need to keep this on a non-main version, this will have a limited blast radius.

Reply:
This one is manageable, yes.